src/python/py_generation_config.cpp

```diff
@@ -111,6 +111,6 @@ void init_generation_config(py::module_& m) {
         .def_readwrite("include_stop_str_in_output", &GenerationConfig::include_stop_str_in_output)
         .def_readwrite("stop_token_ids", &GenerationConfig::stop_token_ids)
         .def_readwrite("adapters", &GenerationConfig::adapters)
-        .def("set_eos_token_id", &GenerationConfig::set_eos_token_id)
+        .def("set_eos_token_id", &GenerationConfig::set_eos_token_id, py::arg("tokenizer_eos_token_id"))
         .def("is_beam_search", &GenerationConfig::is_beam_search);
```

Comment:
BTW, this method is missed https://github.com/openvinotoolkit/openvino.genai/blob/master/src/cpp/include/openvino/genai/generation_config.hpp#L136

Reply:
Added.